docs/guides/05-introduction-to-styling.md

```diff
@@ -97,7 +77,7 @@ To color features by the most common category, you can use the expression [`top`
 color: ramp(top($animals, 4),Prism)
 ```
 
-### Style by number
```

Comment:
Also, the naming is the same we followed in the examples, so they relate better. So we should revert these too.

Reply:
Same

> The problem here is that we have lots of levels so things are not looking good in the dev-center. With this approach, I think it is clearer (at least for me). But feel free to update it.

Reply:
👇

> **Form follows function.** We shouldn't compromise the way we teach our product because the styling is not ideal. Let's not change the content, let's improve the styling.